app/models/recommended_item.rb

```diff
@@ -4,4 +4,8 @@ class RecommendedItem < ActiveRecord::Base
 
   delegate :title, to: :television_show, prefix: true
   delegate :username, to: :user
+
+  def self.newest_first
+    order(created_at: :desc)
```

Comment:
If I believed in default scopes, I would suggest this should be a default scope.

> "Slippery slopes, default scopes", the latest Nü-Jazz album from Ruby Developer, Adarsh Pandit.